test/jdk/java/lang/RuntimeTests/loadLibrary/aix/LoadAIXLibraryFromArchiveObject.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, IBM Corporation. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 8319516
+ * @requires os.family == "aix"
+ * @run  main/othervm -Djava.library.path=/test/lib LoadAIXLibraryFromArchiveObject
+ */
+
+import java.io.*;
+public class LoadAIXLibraryFromArchiveObject {
+    public static void main(String[] args) throws Exception {
+        String libraryName = "awt";
+        File awtSharedObjectPath = new File("/test/lib/libawt.so");
+        File awtArchivePath = new File("/test/lib/libawt.a");
```

Comment:
How does this work? Did you create a "/test" directory? I don't have it on my machine.

Reply:
I am not 100% sure. I think it is a temporary directory under jtreg. I referred to the other test under loadLibrary. @mlchung  Could you clarify this ?